ddl/column.go

```diff
@@ -157,9 +222,8 @@ func (d *ddl) onAddColumn(t *meta.Meta, job *model.Job) (ver int64, _ error) {
 			return ver, errors.Trace(err)
 		}
 
-		// Adjust column offset.
-		d.adjustColumnOffset(tblInfo.Columns, tblInfo.Indices, offset, true)
-		columnInfo.State = model.StatePublic
+		d.adjustColumnOffset(tblInfo.Columns, tblInfo.Indices, 0, lastColOffset, true)
```

Comment:
I think separate `adjustColumnOffset` into three functions is better:
1. `adjustColumnOffsetForDrop([]*model.ColumnInfo, []*model.IndexInfo, int)`;
2. `adjustColumnOffsetForAdd([]*model.ColumnInfo, []*model.IndexInfo, int)`;
3. `adjustColumnOffsetForMultiAdd([]*model.ColumnInfo, []*model.IndexInfo)`;

In fact,  **multi-add** can only append columns but not insert any new columns, the `adjustColumnOffsetForMultiAdd` don't need to do anything.

what do you think?